python/tchannel/tornado/connection.py

```diff
@@ -44,51 +51,97 @@
 
 
 class TornadoConnection(object):
-    """Handle speaking TChannel over a Tornado connection."""
+    """Manages a bi-directional TChannel conversation between two machines.
+
+    The following primary bi-directional operations are provided:
+
+    ``write(message)``
+        Send the message up the wire.
+    ``await(message)``
+        Receive a message.
+    ``send(message)``
+        Send a message and receive its response.
+
+    In addition to those, the following operations are provided and should be
+    used depending on the direction of the connection.
+
+    ``initiate_handshake``
+        Perform a handshake with the remote host.
+    ``expect_handshake``
+        Expect a handshake request from the remote host.
+    """
+
+    # Message ID 0xffffffff is reserved
+    MAX_MESSAGE_ID = 0xfffffffe
```

Comment:
What do you think about moving all constants to single module so that it can be shared between all implementations, not just tornado. Suggest `tchannel.glossary.MAX_MESSAGE_ID`.